components/properties/src/code_point_map.rs

```diff
@@ -358,4 +358,12 @@ pub trait EnumeratedProperty: crate::private::Sealed + TrieValue {
     const NAME: &'static [u8];
     /// The abbreviated name of this property, if it exists, otherwise the name
     const SHORT_NAME: &'static [u8];
+
+    /// Convenience method for `CodePointMapData::new().get(ch)`
```

Comment:
nit: add intra doc links, here and below

```suggestion
    /// Convenience method for [`CodePointMapData::new().get(ch)`](CodePointMapData::get)
```